recipes-multimedia/gstreamer/gstreamer1.0-plugins-good_1.24.7.imx.bb

```diff
@@ -94,11 +94,11 @@ FILES:${PN}-equalizer += "${datadir}/gstreamer-1.0/presets/*.prs"
 DEFAULT_PREFERENCE = "-1"
 
 LIC_FILES_CHKSUM = " \
-    file://LICENSE.txt;md5=a6f89e2100d9b6cdffcea4f398e37343 \
+    file://LICENSE.txt;md5=69333daa044cb77e486cc36129f7a770 \
     file://gst/replaygain/rganalysis.c;beginline=1;endline=23;md5=b60ebefd5b2f5a8e0cab6bfee391a5fe \
 "
 # Enable pulsesink in gstreamer
-PACKAGECONFIG:append = "${@bb.utils.contains('DISTRO_FEATURES', 'pulseaudio', 'pulseaudio', '', d)}"
+PACKAGECONFIG:append = " pulseaudio"
```

Comment:
This is a wrong change because that would cause this feature to be enabled independently if it is supported by the distribution or not.

Reply:
```suggestion
PACKAGECONFIG:append = "${@bb.utils.contains('DISTRO_FEATURES', 'pulseaudio', 'pulseaudio', '', d)}"
```